### ISSUE TITLE
Add Tier-2 experiments for TIPMIP 

### ISSUE THREAD

**Issue by klauswyser** [WCRP-CMIP/CMIP6Plus_CVs#87]
# General Issue: add Tier-2 experiments for TIPMIP project

For the TIPMIP porject we need to save new, non-CMIP6 experiments. A first set of Tier-1 experiments has been discussed in #78 and associated PR #85 . Here is a similar request for Tier-2 experiments, a couple of additional GWL simulations that we are running as part of OptimESM and/or TipESM. @jprb-walton, do you think these are all extra runs, or do you have any other experiments?

| experiment id | activity id | experiment | tier | sub experiment id | parent experiment id | required model components | additional allowed model components | start year | end year | min number of years | parent activity id | description
|-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
| esm-up2p0-gwl1p5 | TIPMIP | zero CO2 emissions simulation starting at global warming level of 1.5K, branching off from esm-up2p0 | 2 | none | esm-up2p0 | AOGCM BGC | AER CHEM |   |   | 300 | TIPMIP | stabilisation at GWL1.5  
| esm-up2p0-gwl3p0 | TIPMIP | zero CO2 emissions simulation starting at global warming level of 3K, branching off from esm-up2p0 | 2 | none | esm-up2p0 | AOGCM BGC | AER CHEM |   |   | 300 | TIPMIP | stabilisation at GWL3
| esm-up2p0-gwl5p0 | TIPMIP | zero CO2 emissions simulation starting at global warming level of 5K, branching off from esm-up2p0 | 2 | none | esm-up2p0 | AOGCM BGC | AER CHEM |   |   | 300 | TIPMIP | stabilisation at GWL5

Colin confirms that we can add these extra GWL exps in the protocol paper:
>Yes, I think we are within our rights to class
>1.	Zero emission runs at GWL=0K (PI), 2K and 4K as TIPMIP Stream/Tier 1.
>2.	Zero emission runs at GWL = 1.5, 3 and 5K as TIPMIP Stream/Tier 2.
>
>Tier 1 will be “required”, Tier 2 will be “encouraged”, but all can be official TIPMIP runs, so valid on the ESGF.
>We can word it this way in the protocol paper
>
  
Pinging @matthew-mizielinski, @jprb-walton, @antworteffekt

**Comment by klauswyser** [comment]
Note that the above list is just a first set of Tier-2 experiments for TIPMIP, there will be more experiments added later. The discussion about Tier-2 experiments is still on-going in the community, unfortunatly only with low intensity so it may take a while before the list Tier-2 experiments can be finalised. However, the GWL simulations mentioned are ready to be added and modelling centres are already doing the simulations and wait for CMIP tables in order to cmorise their output. Could we process this PR quickly (after #85 of course), and then open a new issue for more Tier-2 expeiments later?

**Comment by matthew-mizielinski** [comment]
Added in v6.5.3.0 release
